lib/infra/TranscodingEngineResponse.js

```diff
@@ -3,10 +3,10 @@
  */
 class TranscodingEngineResponse
 {
-	constructor(transcoderResponse, linkToFile)
+	constructor(pathToLog, pathToFile)
 	{
-		this.transcoderResponse = transcoderResponse;
-		this.linkToFile = linkToFile;
+		this.pathToLogFile = pathToLog;
```

Comment:
Add a debug print here - 
"Finish transcoding file $file logs can be found at $log"